src/reducers/reducer.js

```diff
@@ -0,0 +1,22 @@
+const _defaultState = {
+  buttonItems: ["shirt", "shoes", "hat", "sunglasses"],
```

Comment:
good job setting up state - it saved you from writing a lot of boilerplate code!